src/NamingService.ts

```diff
@@ -83,14 +83,14 @@ export default abstract class NamingService extends BaseConnection {
     }
   }
 
-  protected async ignoreResolutionError<T>(
-    code: ResolutionErrorCode | undefined,
+  protected async ignoreResolutionErrors<T>(
+    codes: (ResolutionErrorCode)[],
     promise: Promise<T>,
   ): Promise<T | undefined> {
     try {
       return await promise;
     } catch (error) {
-      if (this.isResolutionError(error, code)) {
+      if (codes.every((code) => this.isResolutionError(error, code))) {
```

Comment:
That is some very slippery code I did. We used to have ignoreResolutionError method and I needed to expand it only for a single-use case. Thought it will do better to change this function rather than creating a separate new one only for this case.